packages/webAuthn/background/webAuthn.ts

```diff
@@ -0,0 +1,226 @@
+import {
+  runtime,
+  tabs,
+  windows,
+  type Runtime,
+  type Tabs,
+  type Windows
+} from "webextension-polyfill"
+
+import json from "~packages/util/json"
+import { PortName } from "~packages/webAuthn/tabs/port"
+import {
+  isWebAuthnError,
+  type WebAuthnAuthentication,
+  type WebAuthnCreation,
+  type WebAuthnError,
+  type WebAuthnParams,
+  type WebAuthnRegistration,
+  type WebAuthnRequest
+} from "~packages/webAuthn/typing"
+
+export const webAuthnCreationAsyncWindow = async (
+  webAuthnCreation?: WebAuthnCreation
+): Promise<WebAuthnRegistration> => {
+  const createWindowUrl = `${runtime.getURL(
+    "tabs/createWebAuthn.html"
+  )}?user=${encodeURI(
+    webAuthnCreation?.user
+  )}&challengeCreation=${webAuthnCreation?.challenge}`
```

Comment:
nit: Maybe we could use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to help constructing the query string.